lib/helper.js

```diff
@@ -3,15 +3,14 @@ var React = require('react/addons');
 var TestUtils = React.addons.TestUtils;
 
 /**
- * Renders a component
+ * Renders a component. Method signature is the same as
+ * https://facebook.github.io/react/docs/top-level-api.html#react.createelement
  *
  * @method renderComponent
- * @param {ReactComponent} Component class
- * @param {Object} props used to initialize the component
  * @return {ReactElement} rendered react element
  **/
-function renderComponent(Component, props) {
-    var element = React.createElement(Component, props);
+function renderComponent() {
+    var element = React.createElement.apply(React, arguments);
```

Comment:
True, but for now it is better to be easier understand because we don't have much documentation...